plugins/outputs/dynatrace/README.md

```diff
@@ -1,53 +1,120 @@
 # Dynatrace Output Plugin
 
-This plugin is sending telegraf metrics to [Dynatrace](https://www.dynatrace.com). It has two operational modes.
+This plugin sends Telegraf metrics to [Dynatrace](https://www.dynatrace.com) via the [Dynatrace Metrics API V2](https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/). It may be run alongside the Dynatrace OneAgent for automatic authentication or it may be run standalone on a host without a OneAgent by specifying a URL and API Token.
 
-Telegraf minimum version: Telegraf 1.16 
-Plugin minimum tested version: 1.16
+## Requirements
+
+You will either need a Dynatrace OneAgent (version 1.201 or higher) installed on the same host as Telegraf; or a Dynatrace environment with version 1.202 or higher. Monotonic counters (e.g. diskio.reads, system.uptime) require Dynatrace 208 or later.
+
+- Telegraf minimum version: Telegraf 1.16
+- Plugin minimum tested version: 1.16
+
+## Getting Started
 
-## Running alongside Dynatrace OneAgent
+Setting up Telegraf is explained in the [Telegraf Documentation](https://docs.influxdata.com/telegraf/latest/introduction/getting-started/).
+The Dynatrace exporter may be enabled by adding an `[[outputs.dynatrace]]` section to your `telegraf.conf`.
+All configurations are optional, but if a `url` is specified then an `api_token` is required.
```

Comment:
```suggestion
All configurations are optional, but if a `url` is specified, then an `api_token` is required.
```
?

Reply:
Is it not possible to export to a OneAgent on a different port? Then the URI would be specified, but no token required.

Reply:
@arminru from https://owl.purdue.edu/owl/general_writing/punctuation/commas/extended_rules_for_commas.html

>However, don't put a comma after the main clause when a dependent (subordinate) clause follows it (except for cases of extreme contrast).

The original version is correct use of commas (english is the worst).

Reply:
Good point, this is covered below somewhere but not consistent with this.

Reply:
@pirgeo I haven't seen any documentation that suggests it is possible. have you seen something like that?

Reply:
https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/#communication-port
https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/telegraf/#communication-port

Reply:
Updated